test/tools/IsolTestOptions.h

```diff
@@ -33,8 +33,10 @@ struct IsolTestOptions: CommonOptions
 	bool noColor = false;
 	bool acceptUpdates = false;
 	std::string testFilter = std::string{};
+	std::string* editor = nullptr;
```

Comment:
```suggestion
	std::string const* editor = nullptr;
```
Maybe?

Reply:
`program_options`  takes a reference to the resources to be modified in response to input on the command line so it can't be a `const std::string*` (or `std::string const*`) because it changes. Did you mean make it a const pointer `std::string * const char`?

Reply:
I see, this works, then.

Reply:
Using a pointer from the outside is a weird thing to do though. Let's refactor it a bit to get rid of the pointer and just store the string in the struct instead.